docs/guides/integration-mobile-client-server.md

```diff
@@ -0,0 +1,377 @@
+---
+title: UID2 Client-Server Integration Guide for Mobile
+sidebar_label: Client-Server Integration for Mobile
+pagination_label: UID2 Client-Server Integration Guide for Mobile
+description: Information about setting up a client-server mobile integration.
+hide_table_of_contents: false
+sidebar_position: 04
+displayed_sidebar: sidebarPublishers
+---
+
+import Link from '@docusaurus/Link';
+
+# UID2 Client-Server Integration Guide for Mobile
+This guide is an overview of integration options for publishers who want to  xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+
+It includes the following sections:
+
+- [Overview](#overview)
+- [Complete UID2 Account Setup](#complete-uid2-account-setup)
+- [Client-Server Mobile Integration Data Flow Overview](#client-server-mobile-integration-data-flow-overview)
+- [Implement Server-Side Token Generation On Your Back-End Server](#xxx)
+- [Server-Side Token Refresh](#server-side-token-refresh)
+- [Add UID2 Mobile SDK into Your Mobile Application](#add-uid2-mobile-sdk-into-your-mobile-application)
+- [Using the UID2 Integration Environment](#using-the-uid2-integration-environment)
+- [Optional: Reduce Latency by Setting the API Base URL for the Production Environment](#optional-reduce-latency-by-setting-the-api-base-url-for-the-production-environment)
+- [Token Storage](#token-storage)
+- [Check that the token was successfully generated and then pass it for bid stream use](#check-that-the-token-was-successfully-generated-and-then-pass-it-for-bid-stream-use)
+- [Best Practice on When to Pass New UID2 Token to the UID2 SDK](#best-practice-on-when-to-pass-new-uid2-token-to-the-uid2-sdk)
+- [Enable Logging (For Android Only)](#enable-logging-for-android-only)
+- [Enable Automatic Token Refresh in Mobile App/Client Side](#enable-automatic-token-refresh-in-mobile-appclient-side)
+- [Optional: UID2 GMA/IMA Plugin for GAM Secure Signal integration](#optional-uid2-gmaima-plugin-for-gam-secure-signal-integration)
+
+This page is intended for the mobile application publishers who want to integrate with UID2 by generating UID2 tokens on their backend servers (or server side) via either a public or private operators and then pass the tokens and user identities into their mobile applications which will in-turn pass the tokens for bid stream use.  
+
+This is called Client-Server Integration because it requires critical integration steps in both client and server side.
+
+If you want to integrate with UID2 via client-side only changes (i.e. all integration changes required are within the mobile applications), refer to the [UID2 Client-Side Integration Guide for Mobile](integration-mobile-client-side.md) instead.
+
+This page provides a high-level overview of integration steps and links to additional documentation.
+
+## Overview
+
+UID2 provides a mobile SDK for Android and iOS with the following features: (**GWH__SW are we now going to have a combo mobile SDK? If we still have two I'll fix the wording.**)
```

Comment:
essentially when i refer to UID2 Mobile SDKs, i just mean UID2 SDK for Android and UID2 SDK for iOS the 2 separate SDKs.

Reply:
Thx. Updated.